pxr/usd/usdLightField/schema.usda

```diff
@@ -0,0 +1,139 @@
+#usda 1.0
+(
+    subLayers = [
+        @usd/schema.usda@,
+        @usdGeom/schema.usda@
+    ]
+)
+
+over "GLOBAL" (
+    customData = {
+        string libraryName       = "usdLightField"
+        string libraryPath       = "pxr/usd/usdLightField"
+        string libraryPrefix     = "UsdLightField"
+        string tokensPrefix      = "UsdLightField"
+    }
+) {
+}
+
+class "GaussiansAPI" (
+    doc = """AppliedAPI schema to extend UsdGeomPoints to describe Gaussians.
+
+    Orientations and Scales are provided in both half and float precision data to allow flexibility
+    in storage. Renderers should prefer the float precision data if present, and they are capable.
+
+    UsdGeomPoints already provides a Widths attribute that represents the size of the point.  In order
+    to allow renderers that do not support rendering Gaussians we suggest setting Widths to the length
+    of the Scales attribute, and then normalizing the Scales attribute.  Renderers reconstruct the
+    original Scales attribute value by multiplying by the UsdGeomPoints Widths attribute.
+    """
+    inherits = </APISchemaBase>
+
+    customData = {
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["UsdGeomPoints"]
+    }
+)  {
+    uniform token gaussianShape = "ellipsoid" (
+        allowedTokens = ["ellipsoid", "triangle", "plane"]
+        doc = """The shape of the gaussian to render.
+        Gaussians may be available in a range of shapes, therefore this list of allowedTokens is not strictly
+        comprehensive. A renderer may support a subset or superset of the allowed tokens.
+
+        An ellipsoid is what many people will colloquially refer to as a splat.
+        """
+    )
+
+    uniform token sortingModeHint = "zDepth" (
+        allowedTokens = ["zDepth", "cameraDistance"]
+        doc = """A hint for the renderer on how to sort the gaussians while drawing.
+        Renderers are free to ignore this, but the hint is often valuable to tune the rendering of the scene.
+
+        Rasterizers usually sort gaussians from back to front.
+        This attribute is a hint for the metric used to sort the gaussians with respect to the camera.
+
+        'zDepth' sorts the gaussians using the z component of the position of the gaussians once
+        transformed in to the cameras projection space.
+
+        'cameraDistance' sorts the gaussians using the euclidean distance between the gaussians
```

Comment:
There's a growing trend of (stochastically) raytracing Gaussian splats as well as works on stochastic rasterization of the splats. In those rendering methods, the Gaussians are sorted **per-pixel** instead of **per-splat** (i.e., only sorted with the center of the splats). This makes a difference, even though both per-pixel and per-splat sorting methods utilize the camera distance (i.e., per-pixel refers to the distance from the camera/ray-origin to the pixel or hit point between the ray and the splat, while per-splat is the distance from the camera to the splat's center, which the ray hits).